css/blog.css

```diff
@@ -0,0 +1,96 @@
+@import url(https://fonts.googleapis.com/css?family=Racing+Sans+One);
```

Comment:
Can you choose the font whic can match the theme

Reply:
I think rail way and open sans better. Didn't?
I'll add it, wait